client/python/samples/get_grpc_model_metadata.py

```diff
@@ -25,20 +25,20 @@
 parser.add_argument('--model_version', default=0, type=int,
                     help='Model version to query. Lists all versions if omitted',
                     dest='model_version')
+parser.add_argument('--timeout', default=10.0, help='Request timeout. default: 10.0',
+                    dest='timeout')
 args = vars(parser.parse_args())
 
 # configuration
 service_url = args.get('service_url')   # default='localhost:9000'
 model_name = args.get('model_name')  # default='resnet'
 model_version = args.get('model_version')   # default=0
+timeout = args.get('timeout')   # default=10.0
 
 # creating grpc client
 client = make_grpc_client(service_url)
 
-# creating metadata request
-request = make_grpc_metadata_request(model_name, model_version)
-
 # getting model metadata from the server
-metadata = client.get_model_metadata(request)
+metadata = client.get_model_metadata(model_name, model_version, timeout)
 metadata_dict = metadata.to_dict()
```

Comment:
Does it work? `get_model_metadata` calls `to_dict()` internally and should return dictionary